worlds/sm/data/sourceinfo.txt

```diff
@@ -1,3 +0,0 @@
-SMBasepatch_prebuilt:
-- comes exactly from build/vanilla/ directory of https://github.com/lordlou/SMBasepatch
-- keep it in sync with the basepatch repo; do not modify the contents in this repo alone!
```

Comment:
You should remove the edits to Super Metroid unless they're somehow required for FF4E